src/webpack/templates/index.html

```diff
@@ -0,0 +1,19 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <%= htmlWebpackPlugin.options.headHtml.join('\n    ') %>
+    <% if (htmlWebpackPlugin.options.rewriteHistory) { %>
+    <script src="/<%= compilation.entrypoints.get('rewriteHistory').getFiles()[0] %>"></script>
+    <% } %>
```

Comment:
Indenting would make this easier to read. does it impact the output?
```suggestion
    <% if (htmlWebpackPlugin.options.rewriteHistory) { %>
      <script src="/<%= compilation.entrypoints.get('rewriteHistory').getFiles()[0] %>"></script>
    <% } %>
```

Reply:
yes it does impact the output

Reply:
funnily enough though, i don't need to flex that so i can remove the if